docsite/index.html

```diff
@@ -2974,6 +2975,58 @@ <h5 style="inline-size: var(--size-header-3)">Carrot cake gummi bears sweet cara
         </div>
       </div>
 
+      <div class="block-wrap">
+        <div>
+          <h5>Relative Sizes</h5>
+          <div class="var-examples">
+            <pre class="language-css"><code>
+              --size-relative-0: .25ch;
+              --size-relative-1: .5ch;
+              --size-relative-2: .75ch;
+              --size-relative-3: 1ch;
+              --size-relative-4: 1.25ch;
+              --size-relative-5: 1.5ch;
+              --size-relative-6: 1.75ch;
+              --size-relative-7: 2ch;
+              --size-relative-8: 3ch;
+              --size-relative-9: 4ch;
+              --size-relative-10: 5ch;
+              --size-relative-11: 6ch;
+              --size-relative-12: 7ch;
+              --size-relative-13: 8ch;
+              --size-relative-14: 10ch;
+              --size-relative-15: 12ch;
+            </code></pre>
+          </div>
+        </div>
+        <div>
+          <h5>Usage Sample</h5>
```

Comment:
be cool if this demo area had a "font size picker" select element so you could see the sizes adapt to a given font size. really help set the feature in. thoughts? it'd be kinda like how line-height and font-size have on.

Reply:
you did perfect 👍🏻